content/en/docs/concepts/architecture/cri.md

```diff
@@ -29,12 +29,8 @@ runtime, which can be configured separately within the kubelet by using the
 `--image-service-endpoint` [command line flags](/docs/reference/command-line-tools-reference/kubelet).
 
 For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
-If a container runtime does not support `v1` of the CRI, then the kubelet tries to
-negotiate any older supported version.
-The v{{< skew currentVersion >}} kubelet can also negotiate CRI `v1alpha2`, but
-this version is considered as deprecated.
-If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
-and doesn't register as a node.
+If a container runtime does not support `v1` of the CRI, then the kubelet cannot register as a node.
+The v{{< skew currentVersion >}} kubelet only supports CRI `v1`. The deprecated `v1alpha2` API is no longer supported. If the kubelet cannot connect to a container runtime using CRI `v1`, it will give up and will not register as a node.
```

Comment:
now these two paragraphs are saying the same thing twice. Also instead of saying "kubelet gives up" please clarify. I think it still registers the node. It just never removes the Not Ready status.

Reply:
Hi @SergeyKanzhelev, It would be great if you could suggest a possible change in the documentation. So that i can directly push those changes.

Reply:
This is basically the task - review docs and update them. Sorry, I only have time to review changes and give a quick feedback. Maybe after code freeze I can find more time.

Reply:
no problem! 
I'll do the needful.